frontend/src/workspace/menu/MakeFunction2.js

```diff
@@ -122,6 +122,10 @@ function FuncGroup(props) {
         ) {
           pos.y = global.height - global.funBarHeight - global.functionWidth;
         }
+        if (pos.x > global.width - 270 - global.functionWidth &&
```

Comment:
These hard-coded numbers make me nervous. What will they do if/when we move the image around?